index.html

```diff
@@ -0,0 +1,192 @@
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
+            
+        </div>
+        <div class="stories1">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+
+    <div class="background-image2">
+        <div class="holder">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
+            
+        </div>
+        <div class="stories">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+
+    <div class="background-image3">
+        <div class="holder1">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
+            
+        </div>
+        <div class="stories1">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+</section>
+<section class="contact-form">
+    <h2>
+        Let's chat!
+    </h2>
+    <p>
+        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it. 
+    </p>
+    <form  action="https://formspree.io/f/mnqwoyza" method="POST">
+        <input id="name" name="Name" class="text" type="text" placeholder="Full Name" maxlength="30" required>
+        <input id="email" name="email" class="text" type="email"  placeholder="example@gmail.com" required>
+        <textarea name="Message" id="message" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+        <button type="button" id="btn" class="button">See Project</button>
+    </form>
+</section>
```

Comment:
- [ ] Please follow the figmal [Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) to match your contact form section.
### **Take A Look**
![form](https://user-images.githubusercontent.com/55596970/167164757-5498645a-69d2-426c-8404-609562627d17.png)
### **Figmal Design**
![fig=form](https://user-images.githubusercontent.com/55596970/167166713-0ea2298c-6565-47d5-badb-4a334262a428.png)